add nested data option for has_one, has_many and moduless params (#14)
phcurado/parameter: README.md

end
```

> Another possibility is avoiding creating files for schema at all. This can be done by importing `Parameter.Schema` and use the `param/2` macro. This is useful for adding params on Phoenix Controllers for example:

Reviewer: ```suggestion
Another possibility is avoiding creating files for a schema at all. This can be done by importing `Parameter.Schema` and using the `param/2` macro. This is useful for adding params in Phoenix controllers. For example:
```

{:error, %{"user_token" => "unknown field"}}
```

> ## Phoenix

Reviewer: what this section is for?
